server/src/test/java/org/elasticsearch/index/IndexSortSettingsTests.java

```diff
@@ -133,46 +118,4 @@ public void testInvalidMissing() {
         assertThat(exc.getMessage(), containsString("Illegal missing value:[default]," +
             " must be one of [_last, _first]"));
     }
-
```

Comment:
The assumptions this test makes no longer hold true, so I've moved it to an IT test above.